nextflow_schema.json

```diff
@@ -10,7 +10,7 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "outdir"],
```

Comment:
Yes when using the flag --show_supported_models, no input is needed, if input is not specified and the flag is not set, it will fail and log accordingly.